aravisApp/src/ADAravis.cpp

```diff
@@ -221,13 +221,14 @@ GenICamFeature *ADAravis::createFeature(GenICamFeatureSet *set,
 static void aravisShutdown(void* arg) {
     ADAravis *pPvt = (ADAravis *) arg;
     GErrorHelper err;
-    ArvCamera *cam = pPvt->camera;
+
     printf("ADAravis: Stopping %s... ", pPvt->portName);
-    arv_camera_stop_acquisition(cam, err.get());
+    arv_camera_stop_acquisition(pPvt->camera, err.get());
```

Comment:
On a general note, passing the error parameter without checking if it NULL first can lead to errors from the Aravis library. I need to check how this is handled elsewhere, but without that, it feels like there's maybe something missing